Tests/Factory/Assembly/TestAssemblies/TyphoonConfigAssembly.m

```diff
@@ -30,4 +32,13 @@ - (id)anotherKnight
     }];
 }
 
+- (id)blockKnight
+{
+    return [TyphoonBlockDefinition withClass:[Knight class] block:^id{
```

Comment:
Is duplication of `Knight` class reference intentional? I mean, it's being used there and later in `[[Knight alloc] init]`.